examples/example-gateway/config/production.json

```diff
@@ -20,6 +20,9 @@
 	"clients.bar.port": 4001,
 	"clients.bar.ip": "127.0.0.1",
 
+	"clients.SimpleService.port": 4002,
```

Comment:
I imagine we will pull out the service name from thrift or client config, and use it as key to get from the above config, https://github.com/uber/zanzibar/pull/27/files#diff-50f740ccde9b3f6a874ee45ca46b1613R49. We can follow convention here, but a conversion is needed in the middle somewhere, which I am not in favor of.

And the `clients` prefix in `clients.xxxx.port` does not really make sense, it's clearly downstream service configs. In this sense, it's arguably more consistent as is the same as the service name defined in thrift.